src/media/media.controller.spec.ts

```diff
@@ -72,4 +76,21 @@ describe('mediaController', () => {
       expect(res).toEqual(response);
     });
   });
+
+  describe('getFile method', () => {
+    const params: GetMediaDTO = {
+      id: '3389e0a0-f543-4423-9c83-39acac2f3381',
+    };
+
+    it('Should call mediaService.getMedia with the correct arguments', () => {
+      mediaService.getMedia(params.id);
+      expect(mediaService.getMedia).toHaveBeenCalledTimes(1);
+      expect(mediaService.getMedia).toHaveBeenCalledWith(params.id);
+    });
+
+    it('Should return non-empty name of the file with the given id', () => {
+      const fileName: string = mediaService.getMedia(params.id);
+      expect(fileName).not.toEqual('');
+    });
+  });
 });
```

Comment:
#### The tests are incomplete. They lack some important login in the controller method. Please check the following suggested test:
```typescript
it('Should call res.sendFile method with correct arguments', () => {
      // data
      const dto: GetMediaDTO = {
        id: 'test-media-uuid',
      };
      const fullPath = path.resolve('media', '/path-to-file');

      // mocks
      const res = { sendFile: jest.fn() };
      mediaService.getMedia = jest.fn().mockReturnValue('/path-to-file');

      // actions
      mediaController.getMedia(dto, res);

      // assertions
      expect(res.sendFile).toBeCalledWith(fullPath);
    });
```
---
#### Please consider the following points in the suggested code:
1- It's good practice to divide your tests to sections (//data, // mocks, // actions, // assertions)
2- It's preferred to have all what you need in your test encapsulated not shared from outer scope. E.g. (mocked functions, mocked data)